libs/platform/src/lib/components/form/multi-input/multi-input.component.ts

```diff
@@ -65,8 +65,16 @@ export class PlatformMultiInputComponent extends BaseMultiInput implements OnIni
     listTemplateDD: ListComponent;
 
     /** Selected values from the list items. */
+
+    _selected: any[] = [];
+
     @Input()
-    selected: any[] = [];
+    set selected(selectedValue: any[]) {
+        this.value = selectedValue;
+    }
+    get selected(): any[] {
+        return this._selected;
+    }
```

Comment:
Do we need `selected` as an @Input when we already have `value` as an @Input from the parent class?
https://github.com/SAP/fundamental-ngx/blob/83d8d3d2ce78a2d0639c403346cbd3c914809824/libs/platform/src/lib/components/form/multi-input/base-multi-input.ts#L134-L147

Reply:
If we need to keep `selected` then can we just leverage `value` like this?
```ts
@Input()
get selected(): any[] {
    return this.value;
}
set selected(value: any[]) {
   this.value = value;
}
```